ingestion/src/metadata/test_suite/validations/column/base/columnValuesToBeBetween.py

```diff
@@ -0,0 +1,101 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# pylint: disable=invalid-name
+
+"""
+Validator for column values sum to be between test case
+"""
+
+import traceback
+from abc import abstractmethod
+from datetime import datetime
+from typing import Union
+
+from sqlalchemy import Column
+
+from metadata.generated.schema.tests.basic import (
+    TestCaseResult,
+    TestCaseStatus,
+    TestResultValue,
+)
+from metadata.orm_profiler.metrics.registry import Metrics
+from metadata.orm_profiler.orm.registry import is_date_time
+from metadata.test_suite.validations.base_test_handler import BaseTestValidator
+from metadata.utils.logger import test_suite_logger
+from metadata.utils.sqa_like_column import SQALikeColumn
+from metadata.utils.time_utils import convert_timestamp
+
+logger = test_suite_logger()
+
+MIN = "min"
+MAX = "max"
+
+
+class BaseColumnValuesToBeBetweenValidator(BaseTestValidator):
+    """ "Validator for column values sum to be between test case"""
```

Comment:
```suggestion
    """Validator for column values to be between test case"""
```
I think the doc string is not correct on some classes.